### PR TITLE
fix: remove named types as it is a breaking change

### PR DESCRIPTION
**Pull request by davidacevedo**
named types are only supported on typescript 4.5 and up

https://devblogs.microsoft.com/typescript/announcing-typescript-4-5/#type-on-import-names

### CODE REVIEW

File: src/components/Form/FormRow.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ const FormRow = (props) => {
           ? React.Children.map(
               children,
               (child) =>
-                React.isValidElement(child) && React.cloneElement(child, { type: inputElementType })
```
Comment (davidacevedo) [inline src/components/Form/FormRow.js:91]: Bad search and replace

File: src/components/Input/CreditCardNumber.tsx [hunk 2]
```diff
@@ -86,7 +86,7 @@ const CreditCardNumber: FC<CreditCardNumberProps> = ({
   };
 
   /* eslint-disable-next-line  @typescript-eslint/no-unused-vars -- Let's figure out a better way to omit props for this scenario */
-  const { type, value, ...inputProps } = props;
```
Comment (davidacevedo) [inline src/components/Input/CreditCardNumber.tsx:89]: Bad search and replace

File: src/components/Input/FileInput.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ class FileInput extends React.Component {
 
   render() {
     /* eslint-disable-next-line  @typescript-eslint/no-unused-vars -- Let's figure out a better way to omit props for this scenario */
-    const { type, onChange, ...props } = this.props;
```
Comment (davidacevedo) [inline src/components/Input/FileInput.js:14]: Bad

File: src/components/Radio/RadioInput.tsx [hunk 4]
```diff
@@ -8,7 +8,7 @@ type RadioInputProps = {
 };
 
 /* eslint-disable-next-line  @typescript-eslint/no-unused-vars -- Let's figure out a better way to omit props for this scenario */
-const RadioInput: FC<RadioInputProps> = ({ type, children, value, ...props }) => (
```
Comment (davidacevedo) [inline src/components/Radio/RadioInput.tsx:11]: bad

File: src/components/Spinner/Spinner.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Spinner } from 'reactstrap';
 import ApmSpinner from './ApmSpinner';
 
-const SpinnerWrapper = ({ type, ...props }) =>
```
Comment (davidacevedo) [inline src/components/Spinner/Spinner.js:5]: bad

File: src/components/Status/Status.tsx [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
-import { type FontAwesomeAPMProps } from '../Icon/FontAwesomeAPM';
+import { FontAwesomeAPMProps } from '../Icon/FontAwesomeAPM';
 import Icon from '../Icon/Icon';
 
 interface StatusProps extends Omit<FontAwesomeAPMProps, 'name'> {
   type?: 'info' | 'muted' | 'success' | 'danger' | 'warning' | 'none';
   className?: string;
 }
 
-const Status = ({ type = 'none', className, ...props }: StatusProps) => {
```
Comment (davidacevedo) [inline src/components/Status/Status.tsx:11]: bad

**Review by JeremyRH — APPROVED** [review]

**Pull request merged**